cmds/bmp/task_command_test.go

```diff
@@ -69,13 +72,92 @@ var _ = Describe("task command", func() {
 			Expect(validate).To(BeTrue())
 			Expect(err).ToNot(HaveOccurred())
 		})
+
+		Context("validates a bad TaskCommand", func() {
+			BeforeEach(func() {
+				options = cmds.Options{
+					Verbose: false,
+					TaskID:  0,
+					Debug:   false,
+				}
+			})
+
+			It("no task_id", func() {
```

Comment:
change description to something like: `"fails when there are no task_id or task_id is 0"`